addons/mail/tests/discuss/test_toggle_upload.py

```diff
@@ -11,7 +11,7 @@
 class TestToggleUpload(HttpCase):
     def test_upload_allowed(self):
         self.authenticate(None, None)
-        channel = self.env["discuss.channel"].create({"name": "General"})
```

Comment:
Note: all tests that have a `"group_public_id": None` added were tests trying to add non-internal users into internal users channel. This is now properly failing.